### ISSUE TITLE
Duplicate `lang` query param in the url causes `request.language.toLowerCase is not a function` error

### ISSUE THREAD

**Issue by nick-fytros** [mashpie/i18n-node#422]
Hello,

If there is more than one url query parameter that sets the language (`req.query.lang` for example) the `request.language.toLowerCase is not a function` error is thrown.
The error is thrown in:
```js
// a query parameter overwrites all
if (queryParameter && request.url) {
  var urlObj = url.parse(request.url, true);
  if (urlObj.query[queryParameter]) {
    logDebug('Overriding locale from query: ' + urlObj.query[queryParameter]);
    request.language = urlObj.query[queryParameter];

    if (preserveLegacyCase) {
      request.language = request.language.toLowerCase();
    }

    return i18n.setLocale(request, request.language);
  }
 }
```
at the `request.language = request.language.toLowerCase();` line. One solution would be to check if the query param is an array and always take the last element but I would like to hear opinions on it.

**Comment by mashpie** [comment]
fix in PR #442 + regression test (pending)

**Comment by mashpie** [comment]
released as 0.11.0 to npm
